### ISSUE TITLE
Ability to add dictionaries

### ISSUE THREAD

**Issue by TimonVS** [swyphcosmo/vscode-spellchecker#18]
Only built-in dictionaries are supported now. It would be nice if we could configure a path to load the dictionaries from.

**Comment by swyphcosmo** [comment]
This has been on my list of additions for a while. I'll push it to the top and working it into the next release. Thanks.

**Comment by Leedehai** [comment]
Yes I really need this.. many jargons and custom class/function names in markdown documents are flagged with an error. Great tool, though:)

**Comment by swyphcosmo** [comment]
@Leedehai it sounds like you actually want to add words to the dictionary and not full languages. This functionality has been around since the beginning of the plugin. Click on the lightbulb icon that appears over the misspelled word, and select 'Ignore' to add the word to your workspace settings file, and 'Always Ignore' to add the word to you global user settings file. Please let me know if I interpreted your comment incorrectly.

**Comment by Leedehai** [comment]
@swyphcosmo oh thanks!

**Comment by kuro68k** [comment]
Any sign of this happening? I'd like to use the en_GB-ize dictionary. Oxford English, as it is sometimes known. Thanks.

**Comment by swyphcosmo** [comment]
@kuro68k There's currently support for `en_GB-ise`, but not for `en_GB-ize`.

Right now, the easiest way to add a new language is to submit a pull request. #46 is an example where someone added support for Swedish.

**Comment by kuro68k** [comment]
Thanks. I am looking at setting up the PR now, but I see there are two options here. There is the en_GB-ize dictionary and the en_GB-large dictionary. The "large" one contains both -ize and -ise.

So I see three options:

1. Remove the -ise dictionary, replace it with the -large one.
2. Add the -ize dictionary.
3. Add the -ize and -large dictionaries.

-ize is Oxford standard and used in academia. Children are mostly taught -ise and that's what most newspapers use. I suppose from my own point of view I consider -ise to be a mistake if I use it, so separate dictionaries is probably better.

**Comment by kuro68k** [comment]
PR here: https://github.com/swyphcosmo/vscode-spellchecker/pull/75

**Comment by swyphcosmo** [comment]
@kuro68k Thanks for the PR! I'm trying to get a couple other open issues addressed and plan to get a new version uploaded to the market place by the end of the week.
